tests/runtime/data/stackdriver/stackdriver_test_monitored_resource.h

```diff
@@ -0,0 +1,52 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+#define MONITORED_RESOURCE_COMMON_CASE	"["		\
+	"1591649196,"			\
+	"{"				\
+        "\"logging.googleapis.com/monitored_resource\": "		\
+        "{"            \
+            "\"labels\": "        \
+            "{"               \
+                "\"project_id\": \"monitored_resource_project_id\","          \
+                "\"location\": \"monitored_resource_location\","          \
+                "\"testA\": \"valA\""      \
+            "}"       \
+        "}"     \
+	"}]"
+
+#define MONITORED_RESOURCE_PRIORITY_HIGHER_THAN_LOCAL_RESOURCE_ID	"["		\
+	"1591649196,"			\
+	"{"				\
+        "\"logging.googleapis.com/monitored_resource\": "		\
+        "{"            \
+            "\"labels\": "        \
+            "{"               \
+                "\"project_id\": \"monitored_resource_project_id\","          \
+                "\"location\": \"monitored_resource_location\","          \
+                "\"cluster_name\": \"monitored_resource_cluster_name\","          \
+                "\"namespace_name\": \"monitored_resource_namespace_name\","          \
+                "\"pod_name\": \"monitored_resource_pod_name\","          \
+                "\"container_name\": \"monitored_resource_container_name\","          \
+            "}"       \
+        "},"     \
+        "\"logging.googleapis.com/local_resource_id\": \"k8s_container.testnamespace.testpod.testctr\""		\
+	"}]"
+
+#define MONITORED_RESOURCE_PRIORITY_HIGHER_THAN_GCE_INSTANCE	"["		\
+	"1448403340,"                               \
+	"{"				\
+        "\"key_0\": false,"                         \
+        "\"key_1\": true,"                          \
+        "\"key_2\": \"some string\","               \
+        "\"key_3\": 0.12345678,"                    \
+        "\"key.4\": 5000,"                          \
+        "\"END_KEY\": \"JSON_END\""                 \
```

Comment:
It doesn't seem that the JSON ends but I may miss something - wha is this key/value pair for?

Reply:
I copied the test data from json.h, which is the test data for 'resource_gce_instance'. But it seems not necessary. I removed the k/v pairs.